src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/AbstractAppSchemaMockData.java

```diff
@@ -394,7 +396,7 @@ private void copy(String content, String location) {
      * @param featureTypeDir feature type directory
      * @param dataStoreName data store directory name
      */
-    private static void writeInfoFile(
+    private void writeInfoFile(
```

Comment:
The received type name as input should already be complete, consequently the method should stay ``public``.

Reply:
Done.